x-pack/plugins/security_solution/public/common/components/matrix_histogram/embeddable_histogram.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { useKibana } from '../../../../../../../src/plugins/kibana_react/public';
+
+const configs = {
+  alignLnsMetric: 'flex-start',
+  appId: 'securitySolutionUI',
+  attributes: [
+    {
+      dataType: 'security',
+    },
+  ],
+  disableBorder: true,
+  disableShadow: true,
+  customHeight: '100%',
+  isSingleMetric: true,
+  owner: 'securitySolution',
```

Comment:
nit: I think you should use the `APP_ID` from `x-pack/plugins/security_solution/common/constants.ts`